### PR TITLE
[HLS-2173] add reads from jaeger-span-saved

### PR DESCRIPTION
**Pull request by levyoren**

### CODE REVIEW

File: plugin/storage/es/spanstore/reader_test.go [hunk 1]
```diff
@@ -575,6 +582,7 @@ func TestSpanReaderFindIndices(t *testing.T) {
 				indexWithDate(spanIndex, dateLayout, today),
 				indexWithDate(spanIndex, dateLayout, yesterday),
 				indexWithDate(spanIndex, dateLayout, twoDaysAgo),
+				"jaeger-span-saved",
```
Comment (prsnca) [inline plugin/storage/es/spanstore/reader_test.go]: can we put it in some const in the code?

**Review by prsnca — CHANGES_REQUESTED** [review]

File: plugin/storage/es/spanstore/reader_test.go (hunk 1 above)
Comment (nata7che) [reply plugin/storage/es/spanstore/reader_test.go]: +1

Comment (levyoren) [reply plugin/storage/es/spanstore/reader_test.go]: done - thanks.

**Review by nata7che — APPROVED** [review]

File: .github/workflows/ci-unit-tests.yml [hunk 2]
```diff
@@ -23,9 +23,9 @@ jobs:
     - name: Run unit tests
       run: make test-ci
 
-    - name: Upload coverage to codecov
-      uses: codecov/codecov-action@v3.1.0
-      with:
-        file: cover.out
-        fail_ci_if_error: true
-        verbose: true
+    # - name: Upload coverage to codecov
```
Comment (prsnca) [inline .github/workflows/ci-unit-tests.yml]: why not just delete this?

Comment (levyoren) [reply .github/workflows/ci-unit-tests.yml]: done.

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+              run: npm install
+
+            - name: Deploy
+              working-directory: scripts
+              run: npx ts-node deploy.ts
+              env:
+                  DOCKER_IMAGE_TAG: ${{ github.event.inputs.dockerImageTag }}
+                  DEPLOY_AWS_ACCESS_KEY: ${{ secrets.DEPLOY_AWS_ACCESS_KEY_ID }}
+                  DEPLOY_AWS_SECRET: ${{ secrets.DEPLOY_AWS_SECRET_ACCESS_KEY }}
+                  AIRBRAKE_PROJECT_ID: 380559
```
Comment (prsnca) [inline .github/workflows/deploy.yml]: you can remove the airbrake params

**Review by prsnca — CHANGES_REQUESTED** [review]

**Review by prsnca — APPROVED** [review]

**Pull request merged**